### ISSUE TITLE
[Bug] Game lags when playing Pong

### ISSUE THREAD

**Issue by CommenterOfComments** [Monika-After-Story/MonikaModDev#1163]
Well, lag isn't really the correct word.

When I play pong with Monika (btw she is some sort of pong god), every time the ball hits a surface, the game freezes for a split second, and time jumps forward when it's done. Happens with both paddles and walls. Also, I swear I saw the ball quite literally teleport through my paddle. Monika had made the ball go into a corner, and the ball just bounced on the wall and then froze, and went past my paddle, despite that being impossible.

Anyone know a fix for this?

**Comment by jmwall24** [comment]
Can't help with lag, but one little tip for Pong is to make sure you are in full screen mode.  If you are in windowed mode, if you go down too far and end up with your cursor off the window, the paddle auto-jumps to the top.  Makes it impossible when it starts going fast.  Once I figured that out I think my record is all the way to like 2-100 lol.

I have noticed sometimes, especially if you are moving the paddle when the ball is hitting it, it does appear that the ball goes thru the paddle.
